toolkit/tools/scheduler/scheduler.go

```diff
@@ -457,6 +476,53 @@ func updateGraphWithImplicitProvides(res *schedulerutils.BuildResult, pkgGraph *
 	return
 }
 
+// setAssociatedDeltaPaths sets the DeltaPath for all of the request's ancillary nodes (both build and run) to the actual
```

Comment:
>DeltaPath [](http://example.com/codeflow?start=36&length=9)

What is a "DeltaPath"? Was a variable name that was removed in one of the commits? This is a bit unclear.

Reply:
Yes, will update to reflect current state. (Originally I didn't want to re-use the existing rpm path variable).